.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
             - /wp-e2e-tests/node_modules
           key: << checksum "package.json" >>
       - run: ./scripts/randomize.sh specs
-      - run: ./scripts/run-wrapper.sh
+      - run: env HEADLESS=1 ./scripts/run-wrapper.sh
```

Comment:
Rather than doing this, why don't we just repurpose the `-x` argument to run.sh to set the envvar?  That way it'll be picked up by all the wrapper repos as well.  It'll also make that argument functional across OSes instead of just on Linux.